apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -264,32 +265,38 @@ const BookingPage = ({
         );
       }
 
-      return router.push({
-        pathname: `/booking/${uid}`,
-        query: {
-          isSuccessBookingPage: true,
-          email: bookingForm.getValues("responses.email"),
-          eventTypeSlug: eventType.slug,
-          seatReferenceUid: "seatReferenceUid" in responseData ? responseData.seatReferenceUid : null,
-          ...(rescheduleUid && booking?.startTime && { formerTime: booking.startTime.toString() }),
-        },
+      const query = {
+        isSuccessBookingPage: true,
+        email: bookingForm.getValues("responses.email"),
+        eventTypeSlug: eventType.slug,
+        seatReferenceUid: "seatReferenceUid" in responseData ? responseData.seatReferenceUid : null,
+        ...(rescheduleUid && booking?.startTime && { formerTime: booking.startTime.toString() }),
+      };
+
+      return bookingSuccessRedirect({
```

Comment:
Do redirection client side which can do redirect outside the iframe. Server can't do that.